assets/scss/modular/_blog-filter.scss

```diff
@@ -0,0 +1,94 @@
+/*
+
+---
+name: Blog Filter
+category:
+ - Components
+ - Components/Blog Filter
+---
+
+```html
+<div class="blog-filter">
+
+	<span>I'd like to see more blogs about <strong>Open Contracting</strong> and </span>
+
+	<span class="blog-filter__options">
+
+		<span class="blog-filter__label">select&nbsp;a&nbsp;filter</span>
+
+		<svg><use xlink:href="#icon-arrow-down"></svg>
+
+		<ul class="blog-filter__list / nav">
+			<li><a href="#">Open Data</a></li>
+			<li><a href="#">Anticorruption</a></li>
+			<li><a href="#">Open Government</a></li>
+		</ul>
+
+	</span>
+
+</div>
+
+<!-- only used to pad out the style guide preview -->
+<div class="band--double-thick">&nbsp;</div>
+```
+
+*/
+
+.blog-filter {
+
+	@include font-size(20);
+	@include font('primary', 'light');
+
+	@include mq(M) {
+		@include font-size(32);
+	}
+
+	strong {
+		@include font('primary');
+	}
+
+}
+
+.blog-filter__options {
+	@include font('primary');
```

Comment:
Hmmm, true, but I feel that it makes it less readable. Sure, with the font weights it's expected that the default weight is "normal" (in most instances), however, I'm not sure that applies in the same way to the font mixin.